modules/config/variables.tf

```diff
@@ -15,18 +15,19 @@ variable "resource_name_prefix" {
 variable "graphdb_admin_password" {
   description = "Password for the 'admin' user in GraphDB."
   type        = string
-  default     = "s3cret"
+  default     = null
+  sensitive   = true
 
-  validation {
-    condition     = length(var.graphdb_admin_password) >= 5 && can(regex("[0-9]", var.graphdb_admin_password))
-    error_message = "Password must be at least 5 characters long and contain at least one number."
-  }
+  # validation {
+  #   condition     = length(var.graphdb_admin_password) >= 5 && can(regex("[0-9]", var.graphdb_admin_password))
```

Comment:
Why?

Reply:
And what if the password is provided by the user? Either we should change the condition or remove it entirely.